frontend/src/utils/api/Groups.ts

```diff
@@ -0,0 +1,11 @@
+import ApiFetch from "../ApiFetch.ts";
+
+export function joinGroup(groupId: number): void {
+    void ApiFetch(`/groups/${groupId}/join`,
```

Comment:
wat doet de void vanvoor in deze functie en de volgende?
als deze weg kan doe je hem weg?

Reply:
Dat kan niet weg, is om aan te duiden dat er met het response van die async functie niets gedaan wordt